app/forms/retention_schedule_form.rb

```diff
@@ -17,6 +20,15 @@ def state_choices
 
   private
 
+  def persist!
```

Comment:
This is a nicer way of doing this than my implementation with a local variable in the function.